Update readme to include official docs link (#2712)
jenssegers/laravel-mongodb: README.md

It is compatible with Laravel 10.x. For older versions of Laravel, please refer to the
[old versions](https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility).

## Documentation

> Reference documentation for the Laravel MongoDB package is available [here](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).

Reviewer: Link on "here" text is not recommended. 

Maybe something like this (not sure):
```suggestion
Read the reference documentation for the Laravel MongoDB package [on MongoDB website](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```

Author: Thanks, makes sense.

> Read the reference documentation for the Laravel MongoDB package [on MongoDB website](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).

Reviewer 2: I'd suggest copying the syntax we already use in https://github.com/mongodb/mongo-php-library/blob/master/README.md#documentation

```md
## Documentation

 - https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/
 - https://www.mongodb.com/docs/drivers/php/
```

Also, I just noticed that https://mongodb.com/docs/ecosystem/drivers/php/ redirects to https://www.mongodb.com/docs/drivers/php/. I'll make that change to the PHPLIB README.

It is compatible with Laravel 10.x. For older versions of Laravel, please refer to the
[old versions](https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility).

## Documentation

> - https://www.mongodb.com/docs/drivers/php/laravel-mongodb/

Reviewer 2: ```suggestion

- https://www.mongodb.com/docs/drivers/php/laravel-mongodb/
```

Line break, consistent with other sections.
